transformer_engine/pytorch/transformer.py

```diff
@@ -435,6 +438,12 @@ def forward(
         rotary_pos_emb: Union[torch.Tensor, Tuple[torch.Tensor, torch.Tensor]], default = `None`
                        Embeddings for query and key tensors for applying rotary position
                        embedding. By default no input embedding is applied.
+        core_attention_bias_type: str, default = `no_bias`
+                    Bias type, {`no_bias`, `pre_scale_bias`, 'post_scale_bias`}
+        core_attention_bias: Optional[torch.Tensor], default = `None`
+                    Bias tensor for Q * K.T
+        fast_zero_fill: bool, defautl = `True`
```

Comment:
```suggestion
        fast_zero_fill: bool, default = `True`
```

Reply:
Why is this arg called `fast_zero_fill`?

Reply:
Done.